app/controllers/admin/dashboard_controller.rb

```diff
@@ -5,4 +5,25 @@ def index
   def show_users
     @users = User.all
   end
+
+  def show_merchants
+    @enabled_merchants = Merchant.where(status: 1)
+    @disabled_merchants = Merchant.where(status: 0)
+  end
+
+  def disable_merchant
+    @merchant = Merchant.find(params[:id])
+    @merchant.update(status: 0)
+    @merchant.items.update(active?: false)
+    flash[:success] = "#{@merchant.name} has been disabled."
+    redirect_to '/admin/merchants'
+  end
+
+  def enable_merchant
+    @merchant = Merchant.find(params[:id])
+    @merchant.update(status: 1)
+    @merchant.items.update(active?: true)
+    flash[:success] = "#{@merchant.name} has been enabled."
+    redirect_to '/admin/merchants'
+  end
```

Comment:
Going off of Brian's comment about Skinny controllers and Thicc Models... Should we move this logic to a model? If so.. which?